<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="4" ignored="0" total="4" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Surefire suite" duration-ms="108925" started-at="2018-07-01T22:38:23Z" finished-at="2018-07-01T22:40:12Z">
    <groups>
    </groups>
    <test name="Surefire test" duration-ms="108925" started-at="2018-07-01T22:38:23Z" finished-at="2018-07-01T22:40:12Z">
      <class name="com.qtpselenium.Zoho.project.testcases.LoginTest">
        <test-method status="FAIL" signature="loginTest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.qtpselenium.Zoho.project.testcases.LoginTest@44f75083]" name="loginTest" duration-ms="19848" started-at="2018-07-01T17:38:25Z" data-provider="getTestData" finished-at="2018-07-01T17:38:45Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[jayav.2007@gmail.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[jayasiri89]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[pass]]>
              </value>
            </param>
          </params>
          <exception class="org.openqa.selenium.NoSuchSessionException">
            <message>
              <![CDATA[Session ID is null. Using WebDriver after calling quit()?Build info: version: '3.8.1', revision: '6e95a6684b', time: '2017-12-01T18:33:54.468Z'System info: host: 'JAYANT', ip: '192.168.1.10', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_144'Driver info: driver.version: RemoteWebDriver]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.NoSuchSessionException: Session ID is null. Using WebDriver after calling quit()?Build info: version: '3.8.1', revision: '6e95a6684b', time: '2017-12-01T18:33:54.468Z'System info: host: 'JAYANT', ip: '192.168.1.10', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_144'Driver info: driver.version: RemoteWebDriverat org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:131)at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:601)at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:658)at org.openqa.selenium.remote.RemoteWebDriver.getScreenshotAs(RemoteWebDriver.java:343)at com.qtpselenium.Zoho.project.BaseTest.Testcase_BaseClass.takeScreenshot(Testcase_BaseClass.java:213)at com.qtpselenium.Zoho.project.BaseTest.Testcase_BaseClass.reportFail(Testcase_BaseClass.java:201)at com.qtpselenium.Zoho.project.testcases.LoginTest.loginTest(LoginTest.java:47)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)at java.lang.reflect.Method.invoke(Unknown Source)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:135)at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:193)at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:94)at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:146)at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:373)at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:334)at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:119)at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:407)]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.NoSuchSessionException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest -->
        <test-method status="FAIL" signature="loginTest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.qtpselenium.Zoho.project.testcases.LoginTest@44f75083]" name="loginTest" duration-ms="28544" started-at="2018-07-01T17:38:45Z" data-provider="getTestData" finished-at="2018-07-01T17:39:14Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[jayav.2007@gmail.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[vgdsgds]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[fail]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[no such element: Unable to locate element: {"method":"id","selector":"ztb-profile-image-pre"}  (Session info: chrome=67.0.3396.99)  (Driver info: chromedriver=2.40.565498 (ea082db3280dd6843ebfb08a625e3eb905c4f5ab),platform=Windows NT 10.0.17134 x86_64) (WARNING: The server did not provide any stacktrace information)Command duration or timeout: 0 millisecondsFor documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.htmlBuild info: version: '3.8.1', revision: '6e95a6684b', time: '2017-12-01T18:33:54.468Z'System info: host: 'JAYANT', ip: '192.168.1.10', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_144'Driver info: org.openqa.selenium.chrome.ChromeDriverCapabilities {acceptInsecureCerts: false, acceptSslCerts: false, applicationCacheEnabled: false, browserConnectionEnabled: false, browserName: chrome, chrome: {chromedriverVersion: 2.40.565498 (ea082db3280dd6..., userDataDir: C:\Users\jayav\AppData\Loca...}, cssSelectorsEnabled: true, databaseEnabled: false, handlesAlerts: true, hasTouchScreen: false, javascriptEnabled: true, locationContextEnabled: true, mobileEmulationEnabled: false, nativeEvents: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: XP, platformName: XP, rotatable: false, setWindowRect: true, takesHeapSnapshot: true, takesScreenshot: true, unexpectedAlertBehaviour: , unhandledPromptBehavior: , version: 67.0.3396.99, webStorageEnabled: true}Session ID: 465657bae8d9671479c574bef0a6c395*** Element info: {Using=id, value=ztb-profile-image-pre}]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: no such element: Unable to locate element: {"method":"id","selector":"ztb-profile-image-pre"}  (Session info: chrome=67.0.3396.99)  (Driver info: chromedriver=2.40.565498 (ea082db3280dd6843ebfb08a625e3eb905c4f5ab),platform=Windows NT 10.0.17134 x86_64) (WARNING: The server did not provide any stacktrace information)Command duration or timeout: 0 millisecondsFor documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.htmlBuild info: version: '3.8.1', revision: '6e95a6684b', time: '2017-12-01T18:33:54.468Z'System info: host: 'JAYANT', ip: '192.168.1.10', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_144'Driver info: org.openqa.selenium.chrome.ChromeDriverCapabilities {acceptInsecureCerts: false, acceptSslCerts: false, applicationCacheEnabled: false, browserConnectionEnabled: false, browserName: chrome, chrome: {chromedriverVersion: 2.40.565498 (ea082db3280dd6..., userDataDir: C:\Users\jayav\AppData\Loca...}, cssSelectorsEnabled: true, databaseEnabled: false, handlesAlerts: true, hasTouchScreen: false, javascriptEnabled: true, locationContextEnabled: true, mobileEmulationEnabled: false, nativeEvents: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: XP, platformName: XP, rotatable: false, setWindowRect: true, takesHeapSnapshot: true, takesScreenshot: true, unexpectedAlertBehaviour: , unhandledPromptBehavior: , version: 67.0.3396.99, webStorageEnabled: true}Session ID: 465657bae8d9671479c574bef0a6c395*** Element info: {Using=id, value=ztb-profile-image-pre}at org.testng.Assert.fail(Assert.java:96)at com.qtpselenium.Zoho.project.BaseTest.Testcase_BaseClass.reportFail(Testcase_BaseClass.java:204)at com.qtpselenium.Zoho.project.BaseTest.Testcase_BaseClass.getElement(Testcase_BaseClass.java:123)at com.qtpselenium.Zoho.project.BaseTest.Testcase_BaseClass.click(Testcase_BaseClass.java:87)at com.qtpselenium.Zoho.project.testcases.LoginTest.loginTest(LoginTest.java:51)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)at java.lang.reflect.Method.invoke(Unknown Source)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:135)at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:193)at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:94)at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:146)at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:373)at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:334)at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:119)at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:407)]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest -->
      </class> <!-- com.qtpselenium.Zoho.project.testcases.LoginTest -->
      <class name="com.qtpselenium.Zoho.project.testcases.DealTest">
        <test-method status="PASS" signature="init()[pri:0, instance:com.qtpselenium.Zoho.project.testcases.DealTest@3d0f8e03]" name="init" is-config="true" duration-ms="27" started-at="2018-07-01T17:38:23Z" finished-at="2018-07-01T17:38:23Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- init -->
        <test-method status="PASS" signature="init()[pri:0, instance:com.qtpselenium.Zoho.project.testcases.LoginTest@44f75083]" name="init" is-config="true" duration-ms="1" started-at="2018-07-01T17:38:23Z" finished-at="2018-07-01T17:38:23Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- init -->
        <test-method status="PASS" signature="init()[pri:0, instance:com.qtpselenium.Zoho.project.testcases.LeadTest@6366ebe0]" name="init" is-config="true" duration-ms="0" started-at="2018-07-01T17:38:23Z" finished-at="2018-07-01T17:38:23Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- init -->
        <test-method status="FAIL" signature="createDeal(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:1, instance:com.qtpselenium.Zoho.project.testcases.DealTest@3d0f8e03]" name="createDeal" duration-ms="36239" started-at="2018-07-01T17:39:14Z" data-provider="getDealTestData1" finished-at="2018-07-01T17:39:50Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Y]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Mozilla]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Deal1]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[RATAN TATA]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[3/15/2018]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[Qualification]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Invalid Login]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Invalid Loginat org.testng.Assert.fail(Assert.java:96)at com.qtpselenium.Zoho.project.BaseTest.Testcase_BaseClass.reportFail(Testcase_BaseClass.java:204)at com.qtpselenium.Zoho.project.testcases.DealTest.createDeal(DealTest.java:50)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)at java.lang.reflect.Method.invoke(Unknown Source)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:135)at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:193)at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:94)at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:146)at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:373)at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:334)at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:119)at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:407)]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createDeal -->
        <test-method status="FAIL" signature="createDeal(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:1, instance:com.qtpselenium.Zoho.project.testcases.DealTest@3d0f8e03]" name="createDeal" duration-ms="20991" started-at="2018-07-01T17:39:50Z" data-provider="getDealTestData1" finished-at="2018-07-01T17:40:11Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Y]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Chrome]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Deal2]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[Ruby]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[5/30/2018]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[Value Proposition]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Invalid Login]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Invalid Loginat org.testng.Assert.fail(Assert.java:96)at com.qtpselenium.Zoho.project.BaseTest.Testcase_BaseClass.reportFail(Testcase_BaseClass.java:204)at com.qtpselenium.Zoho.project.testcases.DealTest.createDeal(DealTest.java:50)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)at java.lang.reflect.Method.invoke(Unknown Source)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:135)at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:193)at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:94)at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:146)at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:373)at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:334)at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:119)at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:407)]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createDeal -->
        <test-method status="PASS" signature="quit()[pri:0, instance:com.qtpselenium.Zoho.project.testcases.DealTest@3d0f8e03]" name="quit" is-config="true" duration-ms="1135" started-at="2018-07-01T17:40:11Z" finished-at="2018-07-01T17:40:12Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quit -->
        <test-method status="PASS" signature="quit()[pri:0, instance:com.qtpselenium.Zoho.project.testcases.LeadTest@6366ebe0]" name="quit" is-config="true" duration-ms="91" started-at="2018-07-01T17:40:12Z" finished-at="2018-07-01T17:40:12Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quit -->
        <test-method status="PASS" signature="quit()[pri:0, instance:com.qtpselenium.Zoho.project.testcases.LoginTest@44f75083]" name="quit" is-config="true" duration-ms="57" started-at="2018-07-01T17:40:12Z" finished-at="2018-07-01T17:40:12Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quit -->
      </class> <!-- com.qtpselenium.Zoho.project.testcases.DealTest -->
    </test> <!-- Surefire test -->
  </suite> <!-- Surefire suite -->
</testng-results>
